Compfiles/Imo1970P6.lean

```diff
@@ -30,17 +30,47 @@ def AcuteTriangle (T : Affine.Triangle ℝ Pt) : Prop :=
   ∠ (T.points 2) (T.points 3) (T.points 1) < Real.pi / 2 ∧
   ∠ (T.points 3) (T.points 1) (T.points 2) < Real.pi / 2
 
-problem imo1970_p6
```

Comment:
Please use `problem` instead of `theorem` for the problem statements. That enables [problem extraction](https://github.com/dwrensha/compfiles?tab=readme-ov-file#extracting-problems).